reana_job_controller/config.py

```diff
@@ -21,7 +21,7 @@
         'reana_job_controller.kubernetes_job_manager.KubernetesJobManager'
     ),
     'htcondorcern': lambda: import_string(
-        'reana_job_controller.htcondorcern_job_manager.JobManagerHTCondorCERN'
+        'reana_job_controller.htcondorcern_job_manager.HTCondorJobManagerCERN'
```

Comment:
- Fixes the problem, runs well on HTC
- The HPC problem still persist; we can fix later